### PR TITLE
chore(ui): Move tailwind config from packages into platform folder

### PR DESCRIPTION
**Pull request by pedrottimark**
### Procedure

After you pull changes, do the following with **stackrox** as current working directory:

```sh
rm ui/packages/tailwind-config/tailwind.css
rm -rf ui/packages/tailwind-config/node_modules

rm -rf ui/node_modules
rm -rf ui/apps/platform/node_modules
```

And then with **stackrox/ui** as current working directory:

1. `yarn`
2. `yarn build`
3. `yarn lint`
4. `yarn start`

Also, restart code editor.

### Problems

1. The amount of tailwind has decreased in Platform UI and some tailwind defaults might conflict with PatternFly or browser defaults.
2. Overlap of devDependencies between packages subfolders and apps/platform cause confusion.

### Analysis

1. This internal dependency builds tailwind **configuration** only. Platform UI builds style bundles.

### Solution

1. Move files from packages/tailwind-config to apps/platform folder.
    * Copy dark.theme.css to src/css folder.
    * Copy light.theme.css to src/css folder.
    * Copy plugins subfolder to apps/platform folder.
    * Merge contents of tailwind.config.js into corresponding file.
2. Add references to copied files in apps/platform folder.
    * Edit src/app.tw.css file.
    * Add tailwindcss forms to devDependencies in apps/platform/package.json file.
3. Delete references to tailwind-config package in ui monorepo.
    * .ossls.yml in stackrox folder.
    * lerna.json in stackrox/ui folder.
    * package.json in stackrox/ui folder.
4. Delete tailwind-config files.

### Residue

1. Move app.tw.css file from src into src/css folder?
2. Delete obsolete plugins and properties from tailwind.config.js file.
3. Investigate which style annoyances are caused by tailwind resets or rules.
4. Investigate reason for `"autoprefixer": "10.4.5"` in resolutions.
5. Remove lerna from devDependencies of ui monorepo.
6. Investigate whether postinstall is still needed for ui monorepo.

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

TL;DR Reduced build time to less than it was before increase for Tailwind purge in #6928

1. `yarn build` in ui and then compute branch - master for the following
    * time -325 = 87 - 412
    * `wc build/static/css/*.css`
        main.css: 2415 = 853478 - 851063
        total: 2415 = 1777785 - 1775370
    * `ls -al build/static/css/*.css | wc -l`
        0 = 33 - 33 files
    * `wc build/static/js/*.js`
        main.css: 0 = 3945375 - 3945375
        total: 0 = 10320471 - 10320471
    * `ls -al build/static/js/*.js | wc -l`
        0 = 89 - 89 files
2. `yarn lint` in ui and ui/apps/platform
3. `yarn test` in ui and ui/apps/platform
4. `yarn start` in ui

### Manual testing

1. Visit /main/dashboard
    * Chrome
    * Firefox
    * Safari

### CODE REVIEW

**Comment by pedrottimark** [comment]
gke-qa-e2e-tests failure possible flake tests

**Review by vjwilson — APPROVED** [review]
